fixtures/input/mdx.mdx

```diff
@@ -0,0 +1,41 @@
+import     {     Baz } from     './Fixture'
+import { Buz  }   from './Fixture'
+
+export  const   foo    = {
+  hi:     `Fudge ${Baz.displayName || 'Baz'}`,
+  authors: [
+     'fred',
+           'sally'
+    ]
+}
+
+# Hello,    world!
+
+
+ I'm an awesome   paragraph.
+
+<!-- I'm a comment -->
```

Comment:
Copied this MDX from Prettier's playground. It seems `eslint-mdx` failed to parse it? Is that a different subset of MDX? @JounQin

Reply:
This is not msx comment.

```mdx
{/* I'm a comment */}
```

Reply:
Even I changed to that, it's still not parsing

Reply:
You can try it by `npm run build` and `npm run test -- -u`

And it seems failed to parse plain markdown with the `====`

Reply:
<img width="238" alt="image" src="https://github.com/antfu/eslint-config/assets/8336744/f47232fa-7f75-4c6d-9c24-6e42a11aa064">


<img width="750" alt="image" src="https://github.com/antfu/eslint-config/assets/8336744/178d9e81-5df3-4526-8977-de783357be32">

Reply:
> And it seems failed to parse plain markdown with the `====`

Such custom markdown syntax requires related remark plugin in `.remarkrc`.

Reply:
It's not custom markdown syntax: https://www.markdownguide.org/basic-syntax/#alternate-syntax

I found out that is because you removed the `parserPlain` in the formatters so there was no parser when you disable markdown plugin but only the formatters. You are right that comment was the only parsing error

Reply:
Good to know.